test/cpp/interop/interop_client.cc

```diff
@@ -451,7 +451,7 @@ bool InteropClient::DoResponseStreaming() {
     // most likely due to connection failure.
     gpr_log(GPR_ERROR,
             "DoResponseStreaming(): Read fewer streams (%d) than "
-            "response_stream_sizes.size() (%" PRIuPTR ")",
+            "response_stream_sizes.size() (%zu)",
```

Comment:
Specifier `z` is not supported in windows. It might be good to avoid using `z` here, though we do not build interop_client in windows. May I know the problem with this message? Maybe we could use another way to fix it.